config/schedule.rb

```diff
@@ -1,17 +1,18 @@
 # Learn more: http://github.com/javan/whenever
 
-def stagger(hour)
-  ('%02d' % hour) + (':%02d' % rand(60)) # some minute between 00 and 59
-end
-
 set :output, 'log/cron.log'
 
-# fortnightly sciencewire harvest at 5pm-ish
-every 2.weeks, at: stagger(17), roles: [:harvester] do
+# fortnightly sciencewire harvest at 5pm in qa, on the 5th and 20th of the month
+every "0 17 5,20 * *", roles: [:harvester_qa] do
+  rake 'sw:fortnightly_harvest'
+end
+
+# fortnightly sciencewire harvest at 5pm in prod, on the 1st and 15th of the month
+every "0 17 1,15 * *", roles: [:harvester_prod] do
   rake 'sw:fortnightly_harvest'
 end
 
-# poll cap for new authorship information nightly at 4am-ish
-every 1.day, at: stagger(4), roles: [:harvester] do
+# poll cap for new authorship information nightly at 4am-ish in both prod and qa
+every 1.day, at: stagger(4), roles: [:harvester_qa, :harvester_prod] do
   rake 'cap:poll[1]'
 end
```

Comment:
Looks like the `stagger` method is still required here on the daily cron.

Reply:
indeed...added back and rebased